model/stream-unit.go

```diff
@@ -10,11 +10,12 @@ import (
 
 type StreamUnit struct {
 	gorm.Model
-	UnitName        string
+
+	UnitName        string `gorm:"not null"`
```

Comment:
I can see this being `null`, most lecturers don't bother thinking about naming their lectures.

Reply:
Ok, will remove this then. :+1: